diff --git tb/chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedSurfaceCoordinator.java tb_diff/chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedSurfaceCoordinator.java
--- tb/chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedSurfaceCoordinator.java
+++ tb_diff/chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedSurfaceCoordinator.java
@@ -4,9 +4,54 @@
 
 package org.chromium.chrome.browser.feed;
 
+import org.chromium.chrome.browser.feed.shared.FeedSurfaceProvider;
+
+import org.chromium.chrome.browser.native_page.ContextMenuManager;
+import org.chromium.chrome.browser.ntp.NewTabPageLayout;
+import org.chromium.components.browser_ui.widget.displaystyle.UiConfig;
+import android.view.View;
+import android.graphics.Canvas;
+
 /**
  * Provides a surface that displays an interest feed rendered list of content suggestions.
  */
-public class FeedSurfaceCoordinator {
+public class FeedSurfaceCoordinator implements FeedSurfaceProvider {
+    public FeedSurfaceCoordinator(Object activity, Object snackbarManager,
+            Object tabModelSelector, Object tabProvider,
+            Object snapScrollHelper, Object ntpHeader,
+            Object sectionHeaderView, Object actionApi,
+            boolean showDarkBackground, Object delegate,
+            Object pageNavigationDelegate, Object profile) {
+    }
+    
     public void destroy() {}
+
+    @Override
+    public ContextMenuManager.TouchEnabledDelegate getTouchEnabledDelegate() {
+        return null;
+    }
+
+    @Override
+    public NewTabPageLayout.ScrollDelegate getScrollDelegate() {
+        return null;
+    }
+
+    @Override
+    public UiConfig getUiConfig() {
+        return null;
+    }
+
+    @Override
+    public View getView() {
+        return null;
+    }
+
+    @Override
+    public boolean shouldCaptureThumbnail() {
+        return false;
+    }
+
+    @Override
+    public void captureThumbnail(Canvas canvas) {
+    }
 }
diff --git tb/chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedUma.java tb_diff/chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedUma.java
new file mode 100644
