diff --git tb/chrome/android/feed/feed_java_sources.gni tb_diff/chrome/android/feed/feed_java_sources.gni
--- tb/chrome/android/feed/feed_java_sources.gni
+++ tb_diff/chrome/android/feed/feed_java_sources.gni
@@ -695,6 +695,17 @@ if (enable_feed_in_chrome) {
     "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedRefreshTask.java",
     "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedSurfaceCoordinator.java",
     "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/TestNetworkClient.java",
+    "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/StreamLifecycleManager.java",
+    "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/NtpStreamLifecycleManager.java",
+    "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/library/api/host/action/ActionApi.java",
+    "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/action/FeedActionHandler.java",
+    "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/v2/FeedStreamSurface.java",
+    "//chrome/android/feed/dummy/java/src/org/chromium/chrome/browser/feed/FeedUma.java",
+    "//chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/library/api/client/stream/Stream.java",
+    "//chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/library/api/client/stream/Header.java",
+    "//chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/shared/FeedSurfaceProvider.java",
+    "//chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/shared/FeedSurfaceDelegate.java",
+    
   ]
   feed_srcjar_deps = []
 }
