diff --git tb/third_party/blink/public/mojom/BUILD.gn tb_diff/third_party/blink/public/mojom/BUILD.gn
--- tb/third_party/blink/public/mojom/BUILD.gn
+++ tb_diff/third_party/blink/public/mojom/BUILD.gn
@@ -210,6 +210,7 @@ mojom("mojom_platform") {
     "//third_party/blink/public/mojom/dom_storage",
     "//third_party/blink/public/mojom/frame",
     "//third_party/blink/public/mojom/usb",
+    "//triple_banana/modules:mojom",
     "//ui/base/cursor/mojom",
     "//ui/base/ime/mojom",
     "//ui/display/mojom",
@@ -295,6 +296,7 @@ mojom("android_mojo_bindings") {
     "//mojo/public/mojom/base",
     "//services/media_session/public/mojom",
     "//services/network/public/mojom",
+    "//triple_banana/modules:mojom",
     "//url/mojom:url_mojom_gurl",
     "//url/mojom:url_mojom_origin",
   ]
